README.md

```diff
@@ -61,6 +61,7 @@ Simply run
 ```sh
 make deploy-full-local-setup
 ```
+> **_NOTE:_** k3d version 4.0 and higher is required, [see](https://k3d.io/usage/configfile/#introduction)
```

Comment:
consider to make note in this section: https://github.com/AbsaOSS/k8gb/blob/master/docs/local.md#environment-prerequisites

Reply:
@kuritka 👍